### ISSUE TITLE
Processing video failed

### ISSUE THREAD

**Issue by ecureuilfourbe** [MacWarrior/clipbucket-v5#232]
Hi when I try to upload a video, it keeps failing at processing the video:

![image](https://user-images.githubusercontent.com/36704066/159955755-4f2bdfe7-1c5a-4247-a075-024e75199e71.png)

and the Conversion log:

[clip.txt](https://github.com/MacWarrior/clipbucket-v5/files/8343161/clip.txt)

Thank you !

**Comment by joo4mart** [comment]
My guess is a problem with the write permission. Does the following path exist:
> /home/http/clipbucket/upload/files/videos/2022/03/24 ?

And which permissions (for writing, which owner) does this directory have?

**Comment by ecureuilfourbe** [comment]
Hi sorry for the late response, 

> Hello @ecureuilfourbe,
> 
> Can you enable development mode and retry ?
> 
> Thanks by advance, Best regards, MacWarrior.

this change nothing.

> My guess is a problem with the write permission. Does the following path exist:
> 
> > /home/http/clipbucket/upload/files/videos/2022/03/24 ?
> 
> And which permissions (for writing, which owner) does this directory have?

The path does exist but there is nothing in it

![image](https://user-images.githubusercontent.com/36704066/160789380-df6e05eb-6a90-4f58-85cb-60aa1af9bbe3.png)

And that's for the permissions

![image](https://user-images.githubusercontent.com/36704066/160789866-fb324419-28de-47df-a703-93e25ad0b788.png)

Thanks

**Comment by MacWarrior** [comment]
Hello @ecureuilfourbe, 

With development mode enable, can you share your video conversion log (Of a video converted with development mode enabled) ?

Best regards,
MacWarrior.

**Comment by MacWarrior** [comment]
Hello @ecureuilfourbe, 

> Unrecognized option 'max_muxing_queue_size'.
What is your version of FFMpeg ?

Best regards,
MacWarrior.

**Comment by ecureuilfourbe** [comment]
The result of the "ffmpeg -version" command

ffmpeg version 2.8.17-0ubuntu0.1

Best regards,

**Comment by MacWarrior** [comment]
Hello @ecureuilfourbe, 

Your issue is here ; Clipbucket require FFMpeg 3+

Best regards,
MacWarrior.

**Comment by ecureuilfourbe** [comment]
Ok so I've upgraded ffmpeg version from 2.8.17 to 4+ and everything works as intended, but I have one last question, is there a way to upgrade ffmpeg via apt command ? I've tried via apt upgrade, update and even a purge/update.

I follow this tutorial:

https://ubuntuhandbook.org/index.php/2020/06/install-ffmpeg-4-3-via-ppa-ubuntu-18-04-16-04/

 but i was still in 2.8.17.

Anyway thanks for taking the time to help me.
Best regards,
